packages/block-library/src/navigation-area/save.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * WordPress dependencies
+ */
+import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
+
+export default function save() {
+	return (
+		<div { ...useBlockProps.save() }>
```

Comment:
I think we can remove the div and only return `InnerBlocks.Content` link the nav block does.

Reply:
I went ahead and did that in af6136d